setup.cfg

```diff
@@ -1,2 +1,44 @@
+[metadata]
+name = astor
+description = Read/rewrite/write Python ASTs
+long_description = file:README.rst
+author = Patrick Maupin
```

Comment:
Can we also add maintainer (`maintainer` and `maintainer-email`) info here?

Reply:
Could you do it yourself? You have commit access, why not to just accept a PR and then add your own commit with anything you want? I just translated the existing setup.py into setup.cfg because I feel it is not right when metadata is stored as code.